third_party/k32w_sdk/k32w_sdk.gni

```diff
@@ -174,11 +176,10 @@ template("k32w_sdk") {
     if (chip_with_se05x == 1) {
       defines += [
         "MBEDTLS_FREESCALE_FREERTOS_CALLOC_ALT=1",
-        "gMainThreadStackSize_c=14048",
+        "CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE=9216",
```

Comment:
Oh and here you set it back to 9k. `chip_with_se05x` is a specific board with more RAM?

Reply:
SE051 is an expansion board that can optionally be attached to the K32W061-DK6 board. It can be used for hardware offloading some crypto operations: https://www.nxp.com/products/security-and-authentication/authentication/edgelock-se051-proven-easy-to-use-iot-security-solution-with-support-for-updatability-and-custom-applets:SE051 When used, a large stack is needed.